extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb

```diff
@@ -161,13 +163,18 @@
                 "\n",
                 "workers_option = f' -w {server_group_workers}' if server_group_workers else \"\"\n",
                 "port_option = f' --port \"{server_group_port}\"' if server_group_port else \"\"\n",
+				"engine_version_option = f' --engine-version -ev {server_group_engine_version}' if server_group_engine_version else \"\"\n",
+				"volume_size_backups_option = f' --volume-size-backups -vsb {server_group_volume_size_backups}' if server_group_volume_size_backups else \"\"\n",
+				"volume_size_data_option = f' volume-size-data -vsd {server_group_volume_size_data}' if server_group_volume_size_data else \"\"\n",
```

Comment:
I don't see where these volume_size_* parameters are defined.  Think they need to be defined above in the "Get optional parameters..." section.